packages/dd-trace/src/profiling/config.js

```diff
@@ -89,6 +106,33 @@ class Config {
 
 module.exports = { Config }
 
+function getExportStrategy (name, options) {
+  const strategy = Object.values(OOMExportStrategies).find(value => value === name)
```

Comment:
We've generally avoided iterator methods as they're often slower than the procedural form. I don't have a strong opinion either way, but just pointing it out.

Reply:
Thanks for pointing this out. This is called only once during startup so I will probably leave it as this.